### ISSUE TITLE
Shadow depending on CraftTweaker?

### ISSUE THREAD

**Issue by ChromaPIE** [GregTechCEu/GregTech#1291]
### GregTech CEu Version

v2.4.4

### Addons Installed

Gregicality Multiblocks v1.1.5
GregTech Food Option v1.8.1

### Environment

SP

### Side Crashed

Client

### New Worlds

Unsure

### Steps to Reproduce

Boot the game with a list of mods shown in the crash report.

### Crash Report

https://hastebin.com/xipukuquqi.rb

### Additional Information

Crash was gone for good with the mods plus CraftTweaker.

**Comment by ALongStringOfNumbers** [comment]
I don't think this is from GTCEu, as I tested a recent build which boot fine without Crafttweaker. However, it could be one of the other addons that are installed. You could try removing those addons one by one and see if the game boots without Crafttweaker.
